### ISSUE TITLE
Android - update guide: duplicated source 

### ISSUE THREAD

**Issue by PietroGranati** [hansemannn/titanium-firebase-analytics#76]
If you set the strings.xml file like in the tutorial:
`<?xml version="1.0" encoding="UTF-8"?>
<resources>
	<string name="google_app_id">YOUR_FIREBASE_APPLICATION_ID</string>
</resources>`

You get this error on compile:

`[INFO]  [GRADLE] > Task :app:mergeDebugResources FAILED
[ERROR] [GRADLE] ERROR:[string/google_app_id] /Users/pietrogranati/Documents/Appcelerator_Studio_Workspace/testu/build/android/app/build/generated/res/google-services/debug/values/values.xml [string/google_app_id] /Users/pietrogranati/Documents/Appcelerator_Studio_Workspace/testu/build/android/app/src/main/res/values/strings.xml: Resource and asset merger: Duplicate resources
[ERROR] [GRADLE] 
[ERROR] [GRADLE] FAILURE: Build failed with an exception.
[ERROR] [GRADLE] 
[ERROR] [GRADLE] * What went wrong:
[ERROR] [GRADLE] Execution failed for task ':app:mergeDebugResources'.
[ERROR] [GRADLE] > [string/google_app_id] /Users/pietrogranati/Documents/Appcelerator_Studio_Workspace/testu/build/android/app/build/generated/res/google-services/debug/values/values.xml      [string/google_app_id] /Users/pietrogranati/Documents/Appcelerator_Studio_Workspace/testu/build/android/app/src/main/res/values/strings.xml: Error: Duplicate resources
[ERROR] [GRADLE] 
[ERROR] [GRADLE] * Try:
[INFO]  [GRADLE] 6 actionable tasks: 1 executed, 5 up-to-date
[ERROR] [GRADLE] > Run with --stacktrace option to get the stack trace.
[ERROR] [GRADLE] > Run with --info or --debug option to get more log output.
[ERROR] [GRADLE] > Run with --scan to get full insights.
[ERROR] [GRADLE] 
[ERROR] [GRADLE] * Get more help at https://help.gradle.org
[ERROR] [GRADLE] 
[ERROR] [GRADLE] BUILD FAILED in 877ms
[ERROR] "gradlew" tool returned exit code: 1
Process exited with 1`

Removing it seems to fix

**Comment by m1ga** [comment]
yes, if you use the json file with gradle (ti > 9) you don't need the strings XML file with the appid
